src/MongoDB.Driver.Core/Core/Authentication/ISecurityContext.cs

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2010-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+namespace MongoDB.Driver.Core.Authentication
+{
+    internal interface ISecurityContext
+    {
+        byte[] Next(byte[] challenge);
+        bool IsInitialized { get; }
```

Comment:
Is this used?

Reply:
`IsInitialized` is used by `GssapiAuthenticator.InitializeStep.Transition` and `GssapiAuthenticator.FirstStep.Transition`.